config/config.go

```diff
@@ -178,13 +178,25 @@ func NewConfig(path string) (*Config, error) {
 		return nil, err
 	}
 
-	err = parseConfig(c.ConfigFile, c.Choria, "", c.rawOpts)
+	err = parseConfig(path, c.Choria, "", c.rawOpts)
 	if err != nil {
 		return nil, err
 	}
 
 	c.parseAllDotCfg()
 
+	for _, path := range ProjectConfigurationFiles(os.Getwd()) {
```

Comment:
This should probably check InitiatedByServer == false.

Reply:
That definitely makes sense!  Fixing this.